### PR TITLE
Del jquery-ujs from webpack.client.base.config.js

### PR DESCRIPTION
**Pull request by justin808**
jquery-ujs is not needed for the Webpack Dev Server setup.

### CODE REVIEW

**Comment by justin808** [comment]
I reported the lint issue: https://github.com/yannickcr/eslint-plugin-react/issues/345

**Comment by justin808** [comment]
@josiasds  @dylangrafmyre What’s the best way to troubleshoot this: https://travis-ci.org/shakacode/react-webpack-rails-tutorial/builds/95525391

Passes locally for me:

```sh
~/shakacode/react-webpack-rails-tutorial (jquery-ujs-only-rails u=) rake ci                                                                                                                                                                    [22:19:04]
Running Rubocop Linters via `rubocop -S -D .`
rubocop -S -D .
Warning: Deprecated pattern style '/Users/justin/shakacode/react-webpack-rails-tutorial/client/node_modules/**/*' in /Users/justin/shakacode/react-webpack-rails-tutorial/.rubocop.yml
^[[1;9DInspecting 36 files
....................................

36 files inspected, no offenses detected
Running ruby-lint Linters via `ruby-lint app config spec lib`
ruby-lint app config spec lib
Running eslint via `cd client && npm run eslint . -- --ext .jsx,.js`
cd client && npm run eslint . -- --ext .jsx,.js

> react-webpack-rails-tutorial@1.1.0 eslint /Users/justin/shakacode/react-webpack-rails-tutorial/client
> eslint --ext .js,.jsx . "." "--ext" ".jsx,.js"

Running jscs via `cd client && npm run jscs .`
cd client && npm run jscs .

> react-webpack-rails-tutorial@1.1.0 jscs /Users/justin/shakacode/react-webpack-rails-tutorial/client
> jscs --verbose . "."

Completed running all JavaScript Linters
scss-lint found no lints
Completed all linting
Running security audit on gems (bundle_audit)
Running security audit on code (brakeman)
brakeman --exit-on-warn --quiet -A -z

+BRAKEMAN REPORT+

Application path: /Users/justin/shakacode/react-webpack-rails-tutorial
Rails version: 4.2.5
Brakeman version: 3.1.2
Started at 2015-12-07 22:56:12 -1000
Duration: 0.208044 seconds
Checks run: BasicAuth, ContentTag, CreateWith, CrossSiteScripting, DefaultRoutes, Deserialize, DetailedExceptions, DigestDoS, EscapeFunction, Evaluation, Execute, FileAccess, FileDisclosure, FilterSkipping, ForgerySetting, HeaderDoS, I18nXSS, JRubyXML, JSONEncoding, JSONParsing, LinkTo, LinkToHref, MailTo, MassAssignment, ModelAttrAccessible, ModelAttributes, ModelSerialize, NestedAttributes, NumberToCurrency, QuoteTableName, Redirect, RegexDoS, Render, RenderDoS, RenderInline, ResponseSplitting, SQL, SQLCVEs, SSLVerify, SafeBufferManipulation, SanitizeMethods, SelectTag, SelectVulnerability, Send, SendFile, SessionManipulation, SessionSettings, SimpleFormat, SingleQuotes, SkipBeforeFilter, StripTags, SymbolDoS, SymbolDoSCVE, TranslateBug, UnsafeReflection, UnscopedFind, ValidationRegex, WeakHash, WithoutProtection, XMLDoS, YAMLParsing


+SUMMARY+

+-------------------+-------+
| Scanned/Reported  | Total |
+-------------------+-------+
| Controllers       | 3     |
| Models            | 2     |
| Templates         | 10    |
| Errors            | 0     |
| Security Warnings | 0 (0) |
+-------------------+-------+



PASSED
```

File: client/package.json [hunk 1]
```diff
@@ -64,7 +64,7 @@
     "webpack": "^1.12.8"
   },
   "devDependencies": {
-    "babel-eslint": "^4.1.5",
+    "babel-eslint": "^5.0.0-beta4",
```
Comment (josiasds) [inline client/package.json]: @justin808 version `5.0.0-beta4` uses Babel 6. Maybe this is causing some conflicts? https://github.com/babel/babel-eslint/commit/a4c09a838cdc4b00b3d754b742a09aeb1b8258c1

**Comment by josiasds** [comment]
@justin808 I believe it was a conflict when merging this with #171, where we created the bundles directory. Fixed now.

@dylangrafmyre Thanks for the tip on slack!

**Comment by justin808** [comment]
Thanks @josiasds! Please be sure the webpack dev server mode works as well.

**Pull request merged**